data/scenarios/Tutorials/conditionals.yaml

```diff
@@ -38,7 +38,7 @@ objectives:
         evaluated until needed.  In this case, we want to make sure
         that only the correct branch is evaluated.  To write a value
         of type, say, `{int}`{=type}, we just surround a value of type `int`{=type}
-        in curly braces, like `{3}`.  This is why arguments to `build`
+        in curly braces, like `{3}`{=snippet}.  This is why arguments to `build`
```

Comment:
Snippets are not parsed or type checked, why add this?

Reply:
Ah, I see.  Actually I had not carefully read the convention documented in #1106.  In that case I will remove the extra `{=snippet}` annotations.

Realizing this does bring to mind that we should have another type of annotation for isolated commands, like:
```
`move`{=command}
```

Reply:
@kostmo those commands are valid swarm code though - basically `LeafCode (Syntax (TConst Move))`.

The only issue is with special things like `require` or keywords like `let`/`def`.

Reply:
@kostmo to clarify, `x` will parse _but not type check_ because it is an unknown variable.

Which is sometimes annoying and I might write something like `x`{.new-variable=actor} to get it added to the type checking phase.